### ISSUE TITLE
CREEL_SAMPLES duplicate samples

### ISSUE THREAD

**Issue by kaijagahm** [MFEh2o/db#69]
`cs <- dbTable("CREEL_SAMPLES")`

`cs %>% group_by(sampleID) %>% filter(n() > 1) %>% as.data.frame() # get non-unique samples`

```
         siteID                                                              sampleID dayOfYear
1 FD_BoatLaunch     FD_BoatLaunch_20180723_2130_CR_fishscapesCreel.Interview.20180910       204
2 FD_BoatLaunch FD_BoatLaunch_20180721_2130_CR_fishscapesCreel.Instantaneous.20180910       202
3 FD_BoatLaunch FD_BoatLaunch_20180721_2130_CR_fishscapesCreel.Instantaneous.20180910       202
4 FD_BoatLaunch     FD_BoatLaunch_20180723_2130_CR_fishscapesCreel.Interview.20180910       204
     dateSet dateSample         dateTimeSet      dateTimeSample    crew gear           sampleGroup
1 2018-07-23 2018-07-23 2018-07-23 13:27:00 2018-07-23 21:30:00 CMI, AT   CR fishScapes_2018_creel
2 2018-07-21 2018-07-21 2018-07-21 13:26:00 2018-07-21 21:30:00 CMI, AT   CR fishScapes_2018_creel
3 2018-07-21 2018-07-21 2018-07-21 13:33:00 2018-07-21 21:30:00 CMI, AT   CR fishScapes_2018_creel
4 2018-07-23 2018-07-23 2018-07-23 05:30:00 2018-07-23 21:30:00 CMI, AT   CR fishScapes_2018_creel
  effort effortUnits useEffort           weather comments                             metadataID
1   8.05  creelHours         1              <NA>     <NA>     fishscapesCreel.Interview.20180910
2   7.75  creelHours         1 some_heavy_clouds     <NA> fishscapesCreel.Instantaneous.20180910
3   7.95  creelHours         1 some_heavy_clouds     <NA> fishscapesCreel.Instantaneous.20180910
4  16.00  creelHours         1              <NA>     <NA>     fishscapesCreel.Interview.20180910
                      updateID
1 originalCreelUpdate_20190125
2 originalCreelUpdate_20190125
3 originalCreelUpdate_20190125
4 originalCreelUpdate_20190125
```

Seem to be unique except for the time/effort. Check data sheets.

**Comment by kaijagahm** [comment]
In the Excel sheet that Ashley sent, we see only 7.95 (correct) and 16.00 (incorrect). Where are the other two?

There is an interview from 7/23 where the person says that they drove 5:30 to get to the lake. That seems to have been accidentally taken as the start time of the creel day. We need a data sheet that shows the actual start time.

Boat and trailer counts may serve as the beginning and end times for the interviews. Ask Ashley for data sheets showing the original times, or check Alex's archive for a blank datasheet, then ask her for those.

**Comment by kaijagahm** [comment]
I bet that the 8.05 came from someone else trying to figure out what the 16.00 value was, digging up the actual start value, and ending up with 8.05. If I try to retrace that, get the start time, etc. and end up with 8.05, then we know that's correct and the 16.00 can be deleted.

Still don't know what's up with the 7.75.

**Comment by kaijagahm** [comment]
Ashley says that Alex's scripts wouldn't contain this. She's sure that 13:27 is the correct value for the start time for the 16:00 sample. 

For the 7.75, I think I should go ahead and delete it, after making sure that all other sampleID's are accounted for (in case 7.75 was somehow masking a different one, or something).

**Comment by kaijagahm** [comment]
Checked that all other sampleID's were present, then removed the two incorrect rows.

**Comment by kaijagahm** [comment]
Closed with db version 3.5.0, 9/30/2020
